victor_fake_hardware_interface/src/victor_fake_hardware_interface/minimal_fake_arm_interface.py

```diff
@@ -220,9 +219,11 @@ def arm_motion_command_callback(self, cmd):
         """
 
         with self.input_mtx:
-            self.motion_status_msg.active_control_mode = self.control_mode_interface.control_mode_parameters_status_msg
+            self.motion_status_msg.active_control_mode = \
+                self.control_mode_interface.control_mode_parameters_status_msg.control_mode
             if cmd.control_mode == self.control_mode_interface.control_mode_parameters_status_msg.control_mode:
-                if cmd.control_mode.mode == ControlMode.JOINT_POSITION or cmd.control_mode.mode == ControlMode.JOINT_IMPEDANCE:
+                if cmd.control_mode.mode == ControlMode.JOINT_POSITION or \
```

Comment:
This might be more clearly and concisely written with a `in [...]` statement

Reply:
Agreed. This was just pycharm autoformatting